### ISSUE TITLE
Show status not updating, showing incorrect status

### ISSUE THREAD

**Issue by whipcracker** [pymedusa/Medusa#8194]
**Describe the bug**
As an episode is added, it won't change from "Wanted" to "Downloaded" and adding previous seasons still shows "Skipped" even though I have changed all previous seasons to "Ignored". Rebooting doesn't fix this either. 

**To Reproduce**

**Expected behavior**
Expecting individual episode statuses to change when selected. 

**Screenshots**
If applicable, add screenshots to help explain your [problem.]

**Medusa (please complete the following information):**
 - OS: Windows 10 Pro 64 bit
 - Branch: master
 - Commit: d9c31da063e6c4b4383749e19b27b13693a1b71e
 - Python version: 2.7.16
 - Database version: 44.15
![Screenshot_20200616-071056](https://user-images.githubusercontent.com/15645911/84707314-d4064e80-afa1-11ea-8bf3-9f61f45ee635.png)

**Debug logs (at least 50 lines):**
General > Advanced Settings > Enable debug
<details>

```
<-- Please replace this whole line with your debug logs -->
```
</details>

**Additional context**

**Comment by medariox** [comment]
It is downloaded as you can see for the "Downloaded" status. It is red because it is not part of your allowed or preferred qualities.

**Comment by whipcracker** [comment]
![Screenshot_20200616-081818](https://user-images.githubusercontent.com/15645911/84711637-29465e00-afaa-11ea-9b66-896df6c991f0.png)
![Screenshot_20200616-081914](https://user-images.githubusercontent.com/15645911/84711658-306d6c00-afaa-11ea-9964-fd83efe5ae18.png)

**Comment by whipcracker** [comment]
> It is downloaded as you can see for the "Downloaded" status. It is red because it is not part of your allowed or preferred qualities.

Thanks for that. 

I think the problem is that the show is showing that it's "Skipped" even though all other seasons bar Season 5 are marked as "Ignored"? 

Any idea what causes this? Cheers!

**Comment by medariox** [comment]
Is your `Status for previously aired episodes` set to `Skipped`?

**Comment by whipcracker** [comment]
> Is your `Status for previously aired episodes` set to `Skipped`?

Not certain, but don't recall it being a problem in the past. What if I wanted to mark some as ignored and some as skipped?

**Comment by medariox** [comment]
A problem? It's just an option... when you add a show you are being asked. You can choose a default status or simply choose a different status every time.

**Comment by whipcracker** [comment]
Hey Dario,

The problem I was referring to basically was that when I added the show, I had it originally set to "Skipped", however if I go through and manually change the previous episiodes to "Ignored", it shouldn't show that I have multiple episodes missing. The status at the top of the page was still showing 40 odd episodes as "Skipped", even though I changed their status to "Ignored". I've not had this problem previously.

I figured out how to get Medusa to properly report them as "Ignored" and that was via "Episode Status Management" and changing "Skipped" episodes to "Ignored" and this fixed the problem.

Assuming it's a bug, but unsure how I could replicate this again if that were the case.

Cheers!
